libs/tup-components/src/utils/Utils.module.css

```diff
@@ -0,0 +1,12 @@
+/* Table Placeholder */
+.empty {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.empty > :first-child {
+    margin-top: 10px;
+    padding: 50px;
+    border: 1px solid var(--global-color-primary--dark);
```

Comment:
I think removing these will let the message inherit Core-Styles. They appear to be from #186. Hm... I wonder why @jarosenb added them.

Reply:
<sup>P.S. The `margin-top` and flex stuff on parent should stay. (Core-Styles doesn't manage spacing and layout for messages.)</sup>

Reply:
Ew. Yes. Thanks. Okay, I look into this.

Reply:
Oh, [found it](https://xd.adobe.com/view/cc3dc245-2792-4e10-bdc1-50d2a7d32979-996e/screen/b1728cf1-1c38-45d1-91c3-560b803632cd/specs/). I didn't recognize it without that "ghost table" behind it. Okay, yes. These message tweaks are good. Please retain.